src/components/organisms/tag-skill-subsection.astro

```diff
@@ -0,0 +1,36 @@
+---
+import Tag from '@/atoms/tag.astro';
+import type { Tag as TagData } from '@/types/common';
+import type { IconName } from '@/types/icon';
+
+export interface Props {
+  skills: TagData[];
+}
+
+const { skills } = Astro.props;
+
+const getIconProps = (name?: IconName, color?: string) => {
+  if (name) {
+    return {
+      name,
+      color,
+    };
+  }
+
+  return undefined;
```

Comment:
Każda funkcja zwraca `undefined` jeśli nie pojawi się wewnątrz niej return więc ta linia nie wnosi nic.

Reply:
To jest akurat kwestia eslint'a 😄 

![Screenshot 2022-10-30 at 09 50 43](https://user-images.githubusercontent.com/68154191/198870222-ecc5dff7-b666-4a19-ba4b-6a4f292fb5c0.png)

Reply:
Nieaktualne już po zmianach w tag'u.

Reply:
Eh @KonradSzwarc i ten jego eslint